plugins/GSdx/GSRendererDX.cpp

```diff
@@ -429,12 +435,10 @@ void GSRendererDX::DrawPrims(GSTexture* rt, GSTexture* ds, GSTextureCache::Sourc
 
 	// vs
 
-	GSDeviceDX::VSSelector vs_sel;
-
-	vs_sel.tme = PRIM->TME;
-	vs_sel.fst = PRIM->FST;
-	vs_sel.logz = dev->HasDepth32() ? 0 : m_logz ? 1 : 0;
-	vs_sel.rtcopy = !!rtcopy;
+	m_vs_sel.tme = PRIM->TME;
+	m_vs_sel.fst = PRIM->FST;
+	m_vs_sel.logz = dev->HasDepth32() ? 0 : m_logz;
```

Comment:
Maybe `!dev->HasDepth32() && m_logz`.

Reply:
I already told him that, guess he didn't listen... :P

Reply:
Message must have refracted and missed. :P

Reply:
Blame @ssakash he didn't like that change. I changed it to that at first :stuck_out_tongue: 

> ssakash: I don't know why you're going so deep into that logic while all I asked for is to remove the last ? 1 : 0 part which coverity will complain about